Controllers/ProductController.js

```diff
@@ -28,6 +29,19 @@ const addSingleProduct = async(req, res) => {
         }
 }
 
+const getSingleProduct = async(req, res) => {
+    const connection = await DbService()
+    const SKU = req.params.SKU
```

Comment:
I think this line is where we should call the sanitise function, like we did for addSingleProduct. ie: 
const sku = DataCheckers.sanitiseSku(req.body.SKU)

Reply:
Agreed, Amended